@types/models/deleteAssetDetail.ts

```diff
@@ -0,0 +1,12 @@
+import { ExchangeRateSymbol } from './exchangeRate';
+
+/**
+ * Delete Portfolio AssetDetail(PortfolioAssetId) 객체 Response Interface
+ */
+export interface DeleteAssetDetail {
+  portfolioAssetId: number;
+  count: number;
+  purchasePrice: number;
+  currencyType: ExchangeRateSymbol;
+  assetId: number;
+}
```

Comment:
```suggestion
export interface DeleteAssetDetailModel {
  portfolioAssetId: number;
  count: number;
  purchasePrice: number;
  currencyType: ExchangeRateSymbol;
  assetId: number;
}
```
이렇게 Model 뒤에 붙이시는건 어떻게 생각하시나요??

Reply:
네 한번에 같이 작업해야할 것 같습니다.